getgist/github.py

```diff
@@ -140,6 +146,14 @@ def select_gist(self, allow_none=False):
 
         return self._ask_which_gist(matches)
 
+    def list_gists(self):
+        """ Prints all gists filenames """       
+        for gist in self.get_gists():
+            for gist_file in gist.get("files"):
+                print ("[{}] {}".format(gist.get("description"),gist_file.get('filename')))
```

Comment:
For maintainability purpuses, we should not use `print`, but [`GetGistCommins.output`](https://github.com/cuducos/getgist/blob/fa0dbe084f71f2d4b9d938de3118140a6e561c3b/getgist/__init__.py#L21-L30).